### DISCUSSION TITLE
Discussion about the international Podcasts interview

### DISCUSSION THREAD

**Discussion by LinuxSuRen** [opensource-f2f/episode#182] [General]
Below is a draft plan for the international Podcasts interview. There are some doubts about it, I will invite some people to discuss it.

* What is the majority of listeners of it? Chinese or people from all over the world.
  * Concern from @LinuxSuRen , don't have good enough listening and speaking skills in English. It might be difficult to do the most job. But this is not a blocking issue.
  * Why should we start a new Podcast considering there are already many awesome open-source Podcasts existing? Such as [The Changlog Podcast](https://changelog.com/podcast)
* Let's assume the target listeners are Chinese. Not sure how many potential Chinese listeners are comfortable if we use English as the main language.

https://github.com/opensource-f2f/episode/blob/master/open-it-up.md

Please feel free to leave your comment below. I'd be happy to talk about it.

**Comment by oleg-nenashev** [comment]
I think a new English podcast could be reasonable if and only if it creates a new area and the content. Having English talks in your podcast could provide some insights to the local community, but the impact would be limited. It makes sense if there is a clear follow-up for the community, for example Google summer of code or an upcoming conference in China.

Maybe you could actually consider inverting things and have Chinese open source project maintainers speaking about their projects in English. There is a big open source community and a lot of local projects, but they have limited visibility across the world.

**Reply by LinuxSuRen** [reply]
> if and only if it creates a new area and the content.

It reminds me to make a list of the open source-related Podcasts before there is a meeting discussion.

And thanks for your idea.

**Reply by LinuxSuRen** [reply]
There is a potential solution. We could use English as the primary language. Then provide a translation script. For example, if I wish to let more Chinese could listen to it easily, I will provide a Chinese script. If other language speakers have a similar idea, they could translate it as well.

A plus of this idea is that our listeners could practice their English listening and reading skills.

**Comment by LinuxSuRen** [comment]
We will have a short meeting to discuss this topic. Please feel free to join us.

* Meeting link: https://meeting.tencent.com/dm/WBna7WXMye5r
* Record link: TBD

> If all the attendees are Chinese speakers, then we will talk in Chinese.
